website/docs/r/avs_private_cloud.html.markdown

```diff
@@ -0,0 +1,134 @@
+---
+subcategory: "Avs"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_avs_private_cloud"
+description: |-
+  Manages a Azure Vmware Solution Private Cloud.
+---
+
+# azurerm_avs_private_cloud
+
+Manages a Azure Vmware Solution Private Cloud.
+
+## Example Usage
+
+```hcl
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_avs_private_cloud" "example" {
+  name                = "example-avs-private-cloud"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  sku_name            = "AV36"
+
+  management_cluster {
+    cluster_size = 3
+  }
+
+  network_block      = "192.168.48.0/22"
+  internet_connected = false
+  nsxt_password      = "QazWsx13$Edc"
+  vcenter_password   = "QazWsx13$Edc"
+  tags = {
+    foo = "bar"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Azure Vmware Solution Private Cloud. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Azure Vmware Solution Private Cloud should exist. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `location` - (Required) The Azure Region where the Azure Vmware Solution Private Cloud should exist. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
+
+* `management_cluster` - (Required) A `management_cluster` block as defined below.
+
+* `network_block` - (Required) The block of addresses which should be unique across virtual network in your subscription as well as on-premise. Changing this forces a new Azure Vmware Solution Private Cloud to be created.
```

Comment:
Is this the same as a subnet? is it a cidr?

Reply:
Right. "networkBlock":"192.168.48.0/22".